README.md

```diff
@@ -102,14 +102,16 @@ func main() {
 You will see these logs in your LogDNA dashboard! Make sure to run .Close() when done with using the logger.
 
 ## Tests
-To run a specific test in the provided test suite add your ingestion key under key and run:
+
+Run all tests in the test suite:
+
 ```
-go test -run TestName
+go test ./logger
 ```
 
-To run all the tests in the test suite run:
+Run a specific test:
 ```
-go test -run ''
+go test ./logger -run ^TestLogger_LogWithOptions$
```

Comment:
Could you explain the changes in running go test here?

Reply:
no changes really other than a documentation update

> Could explain that you need to run go build before running any of the tests. Also is it possible to run go test without building a logger?

it doesn't need to be built first, it can be run directly. the main thing is that the `logger` subdirectory needs to be specified in order to do so. we could really just get rid of the subdirectory too though, not sure how you feel about that.